src/gto-cas-login.php

```diff
@@ -531,11 +532,16 @@ function wp_validate_auth_cookie( $cookie = '', $scheme = '' ) {
       $user = \GlobalTechnology\CentralAuthenticationService\CASLogin::singleton()->get_user_by_guid( $guid );
 
       // return the user id or false if the user doesn't exist
-      return is_null( $user ) ? false : $user->ID;
+      $logged_in = is_null( $user ) ? false : $user->ID;
     }
 
-    // not a valid CAS session, so return false
-    return false;
+    if ( false === $logged_in ) {
+      setcookie( 'logged_in_cookie', ' ', time() - YEAR_IN_SECONDS, COOKIEPATH, COOKIE_DOMAIN );
```

Comment:
Just want to confirm I understand this line properly: it sends back a cookie with a past expiration date, which should have the net effect of deleting the cookie. Correct?

Reply:
Correct, You can't actually "delete" a cookie, just expire it. Local testing showed the cookie disappeared.